### PR TITLE
[get_github_release] Use GITHUB_API_TOKEN for default for consistency

### PR DESCRIPTION
**Pull request by chuganzy**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

Other GitHub related actions use `GITHUB_API_TOKEN` but `get_github_release` does not. For consistency I thought it is better to use `GITHUB_API_TOKEN` here too.

### Description

Just use `ENV['GITHUB_API_TOKEN']` as the default value.

### Testing Steps

n/a

### CODE REVIEW

**Comment by eygraber** [comment]
Is this related? https://github.com/fastlane/fastlane/issues/15979

**Comment by chuganzy** [comment]
@eygraber Hi. I looked into the issue but I think it is not related to this PR.

File: fastlane/lib/fastlane/actions/get_github_release.rb [hunk 1]
```diff
@@ -129,6 +129,7 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :api_token,
                                        env_name: "FL_GITHUB_RELEASE_API_TOKEN",
                                        sensitive: true,
+                                       default_value: ENV["GITHUB_API_TOKEN"],
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/get_github_release.rb]:
```suggestion
                                       code_gen_sensitive: true,
                                       default_value: ENV["GITHUB_API_TOKEN"],
                                       default_value_dynamic: true,
```

Good call! Can we also add these too? ^

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Love this change! Just a need to add `code_gen_sensitive ` and `code_gen_sensitive` and then I think we should be good 😊

**Review by joshdholtz — APPROVED** [review]
Looks good to me now! Thanks for fixing this 😊  Hope all is well ❤️

**Comment by chuganzy** [comment]
@joshdholtz Thank you! Yeah, doing good. Hope you are doing well too!

**Pull request merged**